### PR TITLE
Update first_steps.md

### PR DESCRIPTION
**Pull request by scatterthought**
I don't think the "Log into the dashboard with an existing user" section is necessary in the First Steps, since the user is already logged in after creating an administrator and there are no other existing users. It might confuse people thinking that they're missing something, when they're already logged in. I'd suggest cutting this section (is "logging into the dashboard" covered anywhere else?) and skip right to "Initial Settings".

When I try to space out the images, some of them disappear in the preview. I'm not sure what I'm doing wrong.

Signed-off-by: Russell Wong russ@scatterthought.com (github: scatterthought)

### CODE REVIEW

File: tutorials/getting_started/first_steps.md
Comment (Confectrician) [inline tutorials/getting_started/first_steps.md:28]: Please use one line per sentence.

File: tutorials/getting_started/first_steps.md
Comment (Confectrician) [inline tutorials/getting_started/first_steps.md:36]: Please use one line per sentence.

File: tutorials/getting_started/first_steps.md
Comment (Confectrician) [inline tutorials/getting_started/first_steps.md:67]: Please use one line per sentence.

**Review by Confectrician — COMMENTED** [review]
Hey @scatterthought thanks for your contribution and welcome as a first time contributor. 👍 
Your changes lock very valuable and pretty nice already.

I have left some small comments below.
If you have any questions feel free too ping me here. :)

Also may i ask you to add a sign off statement to your initial posting here?
Explanation in our [contribution guidelines](https://github.com/openhab/openhab-docs/blob/gh-pages/CONTRIBUTING.md#sign-your-work).

For first time contributions in the docs its ok to edit the Pull Request description and add a line like this one:

> Signed-off-by: Joe Smith <joe.smith@email.com>

**Comment by scatterthought** [comment]
Thanks, @Confectrician! I've edited my original comment with the sign-off.

Can I ask why we're limiting to one line? I agree with avoiding long paragraphs, but I write web copy professionally and I wouldn't recommend single-line paragraphs. That approach can sometimes cause more confusion if context is lost.

**Comment by Confectrician** [comment]
>Can I ask why we're limiting to one line?

Of course you can. 🙂 

### Background

We are doing this for a better version tracking for the changes.
We can then track each sentence on its own and it way better to review what has changed for maintainers.
Checking one sentence is way easier than checking a whole paragraph each time.
Hopefully you agree with that.

### Now here comes the trick and reason why you don't need to worry about formatting

During the website build every line of text without a blank line as delimiter will be build as one paragraph.
With a blank line as delimiter you will get another paragraph of sentences until we reach the next blank line or a code block, new heading, ...

You can already see it in your edited file:

Here are two sentences with one line for each of them:
image

On the page they will have a normal paragraph without linebreak:
image


> EDIT: I don't think I fully understand how/where to add the sign-off. Sorry, I've never contributed to GitHub before.

We all have been there. 🙂 
I will take care of the rest for this Pull Request.

If you are interested in how to pass the check immediately, i have written a little FAQ.
The DCO check/sign off is a common practice in open source projects, but we have many contributors in the docs that are not coming from an open source coding background, so the entry hurdle is a bit higher here sometimes.

You can find the FAQ [here in our community forum](https://community.openhab.org/t/dco-check-signing-off-with-github-web-editor-explanation/83330?u=confectrician)

**Comment by scatterthought** [comment]
Thanks, Jerome! I appreciate that you always take the time to thoroughly explain what someone is having trouble understanding. The formatting now makes total sense to me.

**Comment by Confectrician** [comment]
You're welcome. 🙂 

That's what i am here for.
It's not just writing, it's also spreading the word and encourage others to contribute too.

**Comment by Confectrician** [comment]
Hey @scatterthought will you add the changes stated above?
I think we are good to go, when these changes are done and could merge your changes into the docs. 🙂

**Comment by scatterthought** [comment]
Hi @Confectrician,

I tried to edit the file and recommit it with the changes. I thought I did the sign-off correctly (per your FAQ), but I still failed the DCO check. Not sure what I did wrong this time.

**Comment by Confectrician** [comment]
Hey @scatterthought probably it still fails from the first commit you made.

But I will take care of it and give you a feedback here, when I am back on a large screen.

**Comment by Confectrician** [comment]
SO i am not 100% sure, but i think the DCO failed for 2 reasons.

- Firstly you first commit without signoff statement like stated above.
- The second is failing too and i think thats because of the ` (github: scatterthought)`

The DCO check tool doesn't like the github handle from my experience.
Did you get an info from our docs to sign off with a github handle?
If yes, we should change that doc.

**Review by Confectrician — APPROVED** [review]
Thanks 👍

**Comment by scatterthought** [comment]
Oops, I just copied the example you gave earlier in this conversation:

"Signed-off-by: Joe Smith joe.smith@email.com (github: github_handle)"

I noticed that it was different from the FAQ, but I didn't think anything of it as I assumed the "Signed-off-by" was the important part.

Thanks for your help and guidance!

Russ

**Comment by Confectrician** [comment]
>Oops, I just copied the example you gave earlier in this conversation:

🤦‍♂️ oh man. Sorry for that.

**Pull request merged**
